src/components/core/styles/mixins/a11y.scss

```diff
@@ -39,3 +39,13 @@
 @mixin scroll-margin-block-start {
   scroll-margin-block-start: var(--sbb-spacing-fixed-10x);
 }
+
+@mixin disable-animation {
```

Comment:
The thought process was: you might want to disable animations because they are a problem for visually impaired users => it solves an a11y problem => add it to a11y.scss 🥲  (I know, it's a bit stretched)

Anyway, where would you have them? Maybe helpers.scss?

Reply:
... which reminds me, should we maybe also consider https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion ? In theory we could apply disable animation mixin then. but needs broader acceptance then just my idea ;) @kyubisation 

So it maybe does fit somehow in a11y if combining with prefers-reduces-motion. or helpers, or a new one called "animation.ts"?

Reply:
We previously talked about disabling animation/setting animation times to 0 when `prefers-reduced-motion` is configured.
But I wonder if this should be done here or already in the design tokens extension?

Reply:
I think we would just have the same problem of having components needing 0 and some needing 0.1ms ?

Reply:
TBD